src/PageLayout/PageLayout.tsx

```diff
@@ -670,20 +723,44 @@ const Pane = React.forwardRef<HTMLDivElement, React.PropsWithChildren<PageLayout
           }}
           sx={(theme: Theme) => ({
             '--pane-min-width': `256px`,
-            '--pane-max-width': `calc(100vw - 511px)`,
+            '--pane-max-width-diff': '511px',
+            '--pane-max-width': `calc(100vw - var(--pane-max-width-diff))`,
             width: resizable
               ? ['100%', null, 'clamp(var(--pane-min-width), var(--pane-width), var(--pane-max-width))']
               : paneWidths[width],
             padding: SPACING_MAP[padding],
-            [`@media screen and (min-width: ${theme.breakpoints[1]})`]: {
-              overflow: 'auto',
-            },
+            overflowY: 'auto',
 
             [`@media screen and (min-width: ${theme.breakpoints[3]})`]: {
-              '--pane-max-width': 'calc(100vw - 959px)',
+              '--pane-max-width-diff': '959px',
             },
           })}
         >
+          {resizable && (
+            <VisuallyHidden>
+              <form onSubmit={handleWidthFormSubmit}>
+                <p id={`${paneId}-input-hint`}>
+                  Use a value between {minPercent}% and {maxPercent}%
+                </p>
+                <label htmlFor={`${paneId}-width-input`}>
+                  Pane width
+                  <input
+                    id={`${paneId}-width-input`}
+                    aria-describedby={`${paneId}-input-hint`}
+                    name="pane-width"
+                    value={widthPercent}
+                    autoCorrect="off"
+                    autoComplete="off"
+                    type="tel"
+                    onChange={event => {
+                      setWidthPercent(event.target.value)
+                    }}
+                  />
+                </label>
```

Comment:
```suggestion
                <label htmlFor={`${paneId}-width-input`}>
                  Pane width
                </label>
                <p id={`${paneId}-input-hint`}>
                  Use a value between {minPercent}% and {maxPercent}%
                </p>
                  <input
                    id={`${paneId}-width-input`}
                    aria-describedby={`${paneId}-input-hint`}
                    name="pane-width"
                    inputmode="numeric"
                    pattern="[0-9]*"
                    value={widthPercent}
                    autoCorrect="off"
                    autoComplete="off"
                    type="text"
                    onChange={event => {
                      setWidthPercent(event.target.value)
                    }}
                  />
```

Slight tweaks here, and this one is on me. We're:

1. Moving the input hint to after the `label`, so the announcement order is the label, and then a clue about how to operate it.
2. Un-nesting the `input` from its corresponding `label`, for some voice control software corner cases. The programatic association is still present via the `for`/`id` pairing.
3. Replacing `type="tel"` with [this guidance from GOV.UK](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/)